From 67a5562f83cd62281b61637ec03681c9e69b0894 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Mon, 1 Jun 2020 14:37:30 +0900
Subject: [PATCH 81/86] amlogic_vim_spifc_fix

change spi flash freq to 104Mhz for VIM2
change name to w25q128
---
 .../boot/dts/amlogic/meson-gxm-khadas-vim2.dts     | 14 +++++++-------
 arch/arm64/boot/dts/amlogic/meson-khadas-vim3.dtsi |  2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
index e803a651c624..1c8081ab71d9 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-khadas-vim2.dts
@@ -338,13 +338,13 @@
 	pinctrl-0 = <&nor_pins>;
 	pinctrl-names = "default";
 
-	w25q32: spi-flash@0 {
-		#address-cells = <1>;
-		#size-cells = <1>;
-		compatible = "winbond,w25q16", "jedec,spi-nor";
-		reg = <0>;
-		spi-max-frequency = <3000000>;
-	};
+    w25q128: spi-flash@0 {
+	#address-cells = <1>;
+	#size-cells = <1>;
+	compatible = "winbond,w25q128fw", "jedec,spi-nor";
+	reg = <0>;
+	spi-max-frequency = <104000000>;
+    };
 };
 
 /* This one is connected to the Bluetooth module */
diff --git a/arch/arm64/boot/dts/amlogic/meson-khadas-vim3.dtsi b/arch/arm64/boot/dts/amlogic/meson-khadas-vim3.dtsi
index 3fe4c6dd042d..773816baf2f1 100644
--- a/arch/arm64/boot/dts/amlogic/meson-khadas-vim3.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-khadas-vim3.dtsi
@@ -360,7 +360,7 @@
 	pinctrl-0 = <&nor_pins>;
 	pinctrl-names = "default";
 
-	w25q32: spi-flash@0 {
+	w25q128: spi-flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "winbond,w25q128fw", "jedec,spi-nor";
-- 
2.17.1

